emsdk.py

```diff
@@ -256,7 +253,8 @@ def vswhere(version):
     if not program_files:
       program_files = os.environ['ProgramFiles']
     vswhere_path = os.path.join(program_files, 'Microsoft Visual Studio', 'Installer', 'vswhere.exe')
-    output = json.loads(subprocess.check_output([vswhere_path, '-latest', '-version', '[%s.0,%s.0)' % (version, version + 1), '-requires', 'Microsoft.VisualStudio.Component.VC.Tools.x86.x64', '-property', 'installationPath', '-format', 'json']))
+    tools_arch = ('ARM64' if ARCH == 'aarch64' else 'x86.x64')
```

Comment:
Can arm64 be lowercase here? 

Should we switch our internal ARCH=aarch64 to ARCH=arm64 maybe?  

No need for brackets around this expression.

Reply:
This is driven by VS component IDs so it should be capital: https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2022.

`aarch64` came from Unix-like systems and the value is used both for Linux, MacOS and Windows. I'd keep it as is unless we wan't to distinguish between different OSes.